features/crashprobe.feature

```diff
@@ -147,20 +147,22 @@ Scenario: Send a message to an object whose memory has already been freed
 # which is not tested here as this would require symbolication
 Scenario: Crash within Swift code
     When I set environment variable "BUGSNAG_API_KEY" to "a35a2a72bd230ac0aa0f52715bbdc6aa"
-    And I configure the app to run on "iPhone8-11.2"
+    And I configure the app to run on "iPhone8-11.3"
     And I crash the app using "SwiftCrash"
     And I relaunch the app
     Then I should receive a request
     And the request is a valid for the error reporting API
     And the exception "message" equals "Unexpectedly found nil while unwrapping an Optional value"
     And the exception "errorClass" equals "Fatal error"
 
-    And the "method" of stack frame 0 starts with "_T0s18_fatalErrorMessages5NeverOs12StaticStringV_A2E4fileSu4lines6UInt32V5"
-    And the "method" of stack frame 1 starts with "_T010iOSTestApp10SwiftCrashC3run"
+    And the "method" of stack frame 0 equals "_T0s18_fatalErrorMessages5NeverOs12StaticStringV_A2E4fileSu4lines6UInt32V5flagstFTf4nnddn_n"
+    And the "method" of stack frame 1 equals "_T0s18_fatalErrorMessages5NeverOs12StaticStringV_A2E4fileSu4lines6UInt32V5flagstF"
+    And the "method" of stack frame 2 equals "_T010iOSTestApp10SwiftCrashC3runyyF"
+    And the "method" of stack frame 3 equals "_T010iOSTestApp10SwiftCrashC3runyyFTo"
```

Comment:
I'm concerned that this step might break in future (even though supposedly the ABI is stable). A similar equality comparison gave different results on my local machine vs Travis, when using XCode 9.3, which is why the current version uses 'starts with'.

Reply:
Expected:
      +++ actual
      @@ -1 +1 @@
      -`"_T0s18_fatalErrorMessages5NeverOs12StaticStringV_A2E4fileSu4lines6UInt32V5flagstFTf4nnddn_n"`
      +"_T0s18_fatalErrorMessages5NeverOs12StaticStringV_A2E4fileSu4lines6UInt32V5flagstFTfq4nnddn_n"